spec/inputs/jms_spec.rb

```diff
@@ -0,0 +1,90 @@
+require "logstash/devutils/rspec/spec_helper"
+require "jms"
+
+def populate(queue_name, content)
```

Comment:
It would be nice to move this method into a common spec_helper, or support_helper file, that then can be used here.